### PR TITLE
Custom background for Andes Checkbox and Andes Radiobutton

### PR DESCRIPTION
**Pull request by vveltri**
## Description
- Added the possibility of choose background color for Andes Checkbox and Andes Radio Button

Try to answer these questions:
- What is it about?
Customize Andes Checkbox and Andes Radio Button
- How does it work?
You can specify the background color of the Andes Checkbox and Andes Radio Button
## Affected component
Is highly probable that this new code affects directly one (or more?) components inside this lib. Tell us who they are!
- Andes Radio Button
- Andes Checkbox 
## RFC Link
If this change was discussed on RFC please paste link here.
## Screenshots / GIFs
This is a UI library, delight us with some stunning images.
- Before
Captura de Pantalla 2021-01-14 a la(s) 01 08 39
- Now
Captura de Pantalla 2021-01-14 a la(s) 00 54 35

## Checks
Are you sure you followed all those steps? If so, repeat after me "I solemnly swear that ...":
   - [ ] I have coded an example inside the Demo App,
   - [ ] I have added proper tests,
   - [ ] I have modified the Changelog.md file,
   - [ ] I have updated GitHub wiki,
   - [ ] I have the explicit approval of one or more members of the UX Team,
   - [ ] I have checked that this code is ObjC compliant.
   - [ ] I have checked that all the new public enums conform to AndesEnumStringConvertible.
## Change type
This is easy, let us know what this code is about:
   - [ ] This code adds a new component
   - [x] This code includes improvements to an existent component
   - [ ] This code improves or modifies ONLY the Demo App.

### CODE REVIEW

File: LibraryComponents/Classes/Core/AndesCheckbox/Config/AndesCheckboxViewConfig.swift [hunk 1]
```diff
@@ -35,6 +36,7 @@ internal struct AndesCheckboxViewConfig {
         self.iconColor = self.type.iconColor
         self.borderColor = type.borderColor
         self.borderSize = type.borderSize
-        self.backgroundColor = type.backgroundColor
+        self.backgroundColor = checkbox.selectedBackgroundColor != nil ? checkbox.status == .selected ? checkbox.selectedBackgroundColor! : type.backgroundColor : type.backgroundColor
```
Comment (joalonsopint) [inline LibraryComponents/Classes/Core/AndesCheckbox/Config/AndesCheckboxViewConfig.swift:39]: Te parece si implementas un guard para que quede un poco más fácil de entender esta línea de código?

File: LibraryComponents/Classes/Core/AndesCheckbox/View/AndesCheckboxDefaultView.swift [hunk 2]
```diff
@@ -83,6 +83,7 @@ class AndesCheckboxDefaultView: UIView, AndesCheckboxView {
     func updateView() {
         clearView()
         self.label.text = config.title
+        checkboxView.backgroundColor = config.buttonBackgroundColor
```
Comment (joalonsopint) [inline LibraryComponents/Classes/Core/AndesCheckbox/View/AndesCheckboxDefaultView.swift:86]: En el método initialize() se está implementando el el estilo del label, te parece mejor si este background también se inicialice ahí?

File: LibraryComponents/Classes/Core/AndesCheckbox/Config/AndesCheckboxViewConfig.swift (hunk 1 above)
Comment (vveltri) [reply LibraryComponents/Classes/Core/AndesCheckbox/Config/AndesCheckboxViewConfig.swift:39]: Dale, me parece perfecto. Lo hago dentro de una función privada entonces para que sea aún más legible.

File: LibraryComponents/Classes/Core/AndesCheckbox/View/AndesCheckboxDefaultView.swift (hunk 2 above)
Comment (vveltri) [reply LibraryComponents/Classes/Core/AndesCheckbox/View/AndesCheckboxDefaultView.swift:86]: Si, totalmente.

Comment (vveltri) [reply LibraryComponents/Classes/Core/AndesCheckbox/View/AndesCheckboxDefaultView.swift:86]: Estuve revisandolo nuevamente y es necesario que esté acá. Esto es por que al ser una propiedad que se settea luego de inicializarse, esta llama a `updateContentView()` en la clase `AndesCheckbox` que a su vez hace un `contentView.update(withConfig: config)` y esta llama a `updateView()`

**Pull request closed without merging**